client/src/components/commonComponents/NavBar/style.js

```diff
@@ -4,50 +4,62 @@ const useStyles = makeStyles((theme) => ({
   header: {
     backgroundColor: '#000000',
     color: 'white',
-    display: 'flex',
-    height: '40px',
-    justifyContent: 'center',
-    alignItems: 'center',
+    fontSize: '16px',
```

Comment:
One thing you could consider doing is to make a file of styling constants - eg if you have all of the font sizes you use in the app exported from one place it will help keep things consistent

Reply:
@okjake have updated it, I put font sizes of Typography component in materialUITheme file.
Can you check if that's what you meant, please? :sweat_smile: